Can’t install @stryker-mutator/mocha-runner with npm v7

I get the following error when I try to install the latest version `4.1.0` of `@stryker-mutator/mocha-runner` with npm v7:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! While resolving: my-project@0.0.0-dev
npm ERR! Found: @stryker-mutator/core@4.1.0
npm ERR! node_modules/@stryker-mutator/core
npm ERR!   dev @stryker-mutator/core@"4.1.0" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @stryker-mutator/core@"~4.0.0" from @stryker-mutator/mocha-runner@4.1.0
npm ERR! node_modules/@stryker-mutator/mocha-runner
npm ERR!   dev @stryker-mutator/mocha-runner@"4.1.0" from the root project
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
```

It seems like `@stryker-mutator/mocha-runner` is specifying a peer dependency to `@stryker-mutator/core` version < `4.1.0`.

--- a/packages/grunt-stryker/package.json
+++ b/packages/grunt-stryker/package.json
@@ -37,7 +37,7 @@
     "@types/node": "^14.0.1"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "4.0.0-beta.4",
+    "@stryker-mutator/core": "~4.1.0",
     "grunt": ">=0.4.5"
   }
 }

--- a/packages/jasmine-runner/package.json
+++ b/packages/jasmine-runner/package.json
@@ -36,7 +36,7 @@
   },
   "homepage": "https://github.com/stryker-mutator/stryker/tree/master/packages/jasmine-runner#readme",
   "peerDependencies": {
-    "@stryker-mutator/core": "~4.0.0",
+    "@stryker-mutator/core": "~4.1.0",
     "jasmine": ">=2"
   },
   "devDependencies": {

--- a/packages/jest-runner/package.json
+++ b/packages/jest-runner/package.json
@@ -51,7 +51,7 @@
     "react-scripts-ts": "~3.1.0"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "~4.0.0",
+    "@stryker-mutator/core": "~4.1.0",
     "jest": ">= 22.0.0"
   },
   "dependencies": {

--- a/packages/karma-runner/package.json
+++ b/packages/karma-runner/package.json
@@ -44,7 +44,7 @@
     "karma-mocha": "~2.0.0"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "~4.0.0"
+    "@stryker-mutator/core": "~4.1.0"
   },
   "dependencies": {
     "@stryker-mutator/api": "4.1.1",

--- a/packages/mocha-runner/package.json
+++ b/packages/mocha-runner/package.json
@@ -48,7 +48,7 @@
     "@types/node": "^14.0.1"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "~4.0.0",
+    "@stryker-mutator/core": "~4.1.0",
     "mocha": ">= 2.3.3 < 9"
   }
 }

--- a/packages/typescript-checker/package.json
+++ b/packages/typescript-checker/package.json
@@ -42,7 +42,7 @@
     "@types/semver": "~7.3.3"
   },
   "peerDependencies": {
-    "@stryker-mutator/core": "~4.0.0",
+    "@stryker-mutator/core": "~4.1.0",
     "typescript": ">=3.6"
   }
 }

